rust/rust.bzl

```diff
@@ -307,7 +320,8 @@ def _rust_library_impl(ctx):
       rust_srcs = ctx.files.srcs,
       rust_deps = ctx.attr.deps,
       transitive_libs = depinfo.transitive_libs,
-      rust_lib = rust_lib)
+      rust_lib = rust_lib,
+      native_link_flags = depinfo.native_link_flags)
```

Comment:
AFAICT it doesn't need to be. Since all the deps are collected and symlinked into a single `.deps` directory there's only one search path for any native dep.